src/jobs/salaryScheduleJob.ts

```
@@ -0,0 +1,46 @@
+    } catch (err) {
+      logger.error("Failed to trigger salary schedule", { scheduleId: schedule.id, error: err });
+    }
+  }
+}
+
+/**
+ * Start the salary schedule scheduler.
+ */
+export async function startSalaryScheduleScheduler(): Promise<void> {
+  const intervalMs = 60 * 1000; // Run every minute
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)